saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -10,21 +10,31 @@
 from ....product.tasks import update_product_minimal_variant_price_task
 from ....product.utils import delete_categories
 from ....product.utils.attributes import generate_name_for_variant
+from ....warehouse import models as warehouse_models
+from ....warehouse.error_codes import StockErorrCode
```

Comment:
It seems it was here before but there is a typo: `StockErorrCode` should be `StockErrorCode`. It would be nice to fix it since we're changing related mutations in this PR.